src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -64,9 +60,8 @@ export const PositionsTriggersCell = ({
   const stringGetter = useStringGetter();
   const dispatch = useAppDispatch();
   const { isSlTpLimitOrdersEnabled } = useEnvFeatures();
-  const { complianceState } = useComplianceState();
 
-  const onViewOrders = isDisabled ? null : () => onViewOrdersClick(marketId);
+  const onViewOrders = () => onViewOrdersClick(marketId);
```

Comment:
small fix: isDisabled = true when account is view only, but user should be able to view orders even if it's view only